app/Resources/views/admin/blog/index.html.twig

```diff
@@ -5,6 +5,8 @@
 {% block main %}
     <h1>{{ 'title.post_list'|trans }}</h1>
 
+    {{ helper.render_flash_messages() }}
```

Comment:
I had some doubts about this. But after reading your comment, I've created the template and removed te macros. We'll add some macros in other feature. Thanks!